Sources/ReactorKit/Reactor.swift

```diff
@@ -145,22 +154,30 @@ extension Reactor {
       })
       .replay(1)
     transformedState.connect().disposed(by: disposeBag)
-    return transformedState
+    return ReactorStreams(action: actionSubject, event: transformedEvent, state: transformedState)
   }
 
   public func transform(action: Observable<Action>) -> Observable<Action> {
     action
   }
 
-  public func mutate(action: Action) -> Observable<Mutation> {
+  public func mutate(action: Action) -> Observable<Event> {
     .empty()
   }
-
-  public func transform(mutation: Observable<Mutation>) -> Observable<Mutation> {
+  
+  public func transform(event: Observable<Event>) -> Observable<Event> {
+    transform(mutation: event)
+  }
+  
+  public func transform(mutation: Observable<Event>) -> Observable<Event> {
```

Comment:
When both `transform(event:)` and `transform(mutation:)` are overridden, the mutation can be ignored.
If we're facing this level of a breaking point, how about removing `transform(mutation:)` and enforcing the use of event instead?

Reply:
> However, I'm concerned about the significant breaking point it introduces for existing users.

completely agree

Reply:
> When both transform(event:) and transform(mutation:) are overridden, the mutation can be ignored.
> If we're facing this level of a breaking point, how about removing transform(mutation:) and enforcing the use of event instead?

Like you said, I think it's okay to move it all at once! Previously, if developers recognized that they needed to define the `transform(event:)` function, it was judged that there was no possibility for them to define `transform(mutation:)` at all. So, I thought, why not set a time when both mutations and events can coexist for developers who want to slowly reflect breaking changes? However, if I don't explain it well enough, it will be difficult to completely ignore the confusion you mentioned, and it's also a hassle to explain.

Reply:
First, look at this PR as a blueprint, and feel free to rework it to fit your new release plan. I think it would be okay to expose the mutation stream first and add a PR that changes the name to event later.

Reply:
I was seriously mistaken about something. I thought that by using event, it would fundamentally follow the operation of Pulse. It turns out it was just a rename.
Nonetheless, as I mentioned before, I'm still concerned about the significant breaking point.

I also think that 'Event' sounds more natural than 'State'. Maybe it's a good idea to just advertise loudly, 'ReactorKit has changed, so please use 'Event' from now on.' 😂

We need to gather opinions from others as well. @OhKanghoon

Reply:
I separated the PRs to make it easier to review and merge.
https://github.com/ReactorKit/ReactorKit/pull/226